package.json

```diff
@@ -48,6 +48,16 @@
                     "default": true,
                     "description": "Adds regions when organizing a class."
                 },
+                "tsco.addRowNumberInRegionName": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Add region children count on title."
+                },
+                "tsco.accessorsBeforeCtor": {
+                    "type": "boolean",
```

Comment:
instead of bool, maybe make array to collect wanted order to better custom the order ?

example : [
   ''prop",
   ''ctor",
   ''accessor",
   ''func",
  ]

It's not my tool, I don't want do a lot of changes without ask you !
But if you like this idea, we can make it together :D

Reply:
Yes, your're right, this would make it more flexible indeed. If you can manage, feel free to implement it and I'll guide you if necessary.